api/client/crop_model.py

```diff
@@ -63,11 +64,16 @@ def mapper(region):
                       (df['metric_id'] == entities['metric_id']) &
                       (df['region_id'] == region['id'])]['value'].mean(skipna=True)
         means = list(map(mapper, regions))
-        self._logger.debug('Means = {}'.format(
+        self.get_logger().debug('Means = {}'.format(
             list(zip([region['name'] for region in regions], means))))
         # Normalize into weights
         total = math.fsum([x for x in means if not math.isnan(x)])
```

Comment:
a minor improvement here since we are using numpy, this is equivalent to `numpy.nansum(mean)`

Reply:
done